index.html

```diff
@@ -4,10 +4,12 @@
     <meta charset="UTF-8">
     <title>Landing</title>
     <link rel="stylesheet" href="css/main.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+    <script src="js/script.js"></script>
   </head>
   <body>
     <div class="wrapper">
-      <div class="top-header">
+      <div class="top-header" id="top-header">
```

Comment:
It's not a best practice to set same name to class and id. It works, but a little bit confusing